core/src/main/java/org/infinispan/commands/irac/IracCleanupKeyCommand.java

```diff
@@ -0,0 +1,110 @@
+package org.infinispan.commands.irac;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.commands.CommandInvocationId;
+import org.infinispan.commands.InitializableCommand;
+import org.infinispan.commands.remote.CacheRpcCommand;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.util.ByteString;
+import org.infinispan.util.concurrent.CompletableFutures;
+import org.infinispan.xsite.irac.IracManager;
+
+/**
+ * // TODO: Document this
+ *
+ * @author Pedro Ruivo
+ * @since 8.0
```

Comment:
8?

Reply:
I need to update the IDEA template xD just ignore it